src/Language/Haskell/Liquid/Types.hs

```diff
@@ -458,8 +458,9 @@ instance Hashable (PVar a) where
 ------ Strictness --------------------------------------------------
 --------------------------------------------------------------------
 
-instance NFData Var
-instance NFData SrcSpan
+instance NFData Var where rnf _ = ()
```

Comment:
So when I added the Werrors flag I came across some real errors, ie not
stylistic warnings, and now that they are fixed I don't want to allow them
again.

Just create a 7.10 branch for the ones that rash to upgrade.
On Mar 27, 2015 3:49 PM, "Eric Seidel" <notifications@github.com> wrote:

> In src/Language/Haskell/Liquid/Types.hs
> <https://github.com/ucsd-progsys/liquidhaskell/pull/364#discussion_r27336650>
> :
>
> > @@ -458,8 +458,9 @@ instance Hashable (PVar a) where
> >  ------ Strictness --------------------------------------------------
> >  --------------------------------------------------------------------
> >
> > -instance NFData Var
> > -instance NFData SrcSpan
> > +instance NFData Var where rnf _ = ()
>
> I disagree, I don't find Werror to be particularly useful in Haskell where
> most of the warnings are cosmetic or stylistic.
>
> In contrast, not everyone will rush to upgrade to 7.10; so moving to 7.10
> will prevent people from using LH.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ucsd-progsys/liquidhaskell/pull/364/files#r27336650>.
>